whipper/command/mblookup.py

```diff
@@ -1,46 +1,57 @@
 from whipper.command.basecommand import BaseCommand
-from whipper.common.mbngs import musicbrainz
+from whipper.common.mbngs import musicbrainz, getReleaseMetadata
 
 
 class MBLookup(BaseCommand):
     summary = "lookup MusicBrainz entry"
-    description = """Look up a MusicBrainz disc id and output information.
+    description = """
+Look up a MusicBrainz disc id or release id and output information.
 
 You can get the MusicBrainz disc id with whipper cd info.
 
-Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-"""
+Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-
+"""
```

Comment:
What are you doing these changes for? They don’t seem to be related to the PR per se.

Reply:
[Few lines above](https://github.com/whipper-team/whipper/pull/432/files#diff-7f0abcaabcda29c0ee0003d36f8c4874R8), i added `disc id and release id` instead of just `discid` and the line was a liitle too long so i looked at other files like [this one](https://github.com/whipper-team/whipper/blob/develop/whipper/command/image.py#L35#L37) to see how to structure it then i made the change.

Reply:
Sure, I’m not saying it’s a bad change—I’m saying it’s a change unrelated to this commit/PR and as such should be its own commit/PR. Check out "[Developer Tip: Keep Your Commits “Atomic”](https://www.freshconsulting.com/atomic-commits/)". :)

Reply:
Ah, okay i get it.